torch/csrc/jit/codegen/cuda/partition.cpp

```diff
@@ -57,6 +57,20 @@ inline bool isFusableNode(const Node* const node) {
   return (isNodeParsible(node) || node->kind() == prim::CudaFusionGroup);
 }
 
+bool hasReductionOperation(const Node* const node) {
+  if (isReductionNode(node)) {
+    return true;
+  }
+  if (node->kind() == prim::CudaFusionGroup) {
+    for (auto n : node->g(attr::Subgraph)->nodes()) {
+      if (hasReductionOperation(n)) {
+        return true;
+      }
+    }
+  }
+  return false;
+}
+
 } // namespace
 
 bool isFusableCudaFusionGroup(const Node* const node) {
```

Comment:
sigh, I see the extra consts are everywhere. I can do a separate pass to clean this up then...

Reply:
I updated all of `const T* const` in integration code, but left the codegen part alone. :stuck_out_tongue: 
I did notice our migration to remove the const pointer. Out of curiosity, what's to hate const pointer?